docs/build/guides/fungible-token.md

```diff
@@ -494,31 +872,34 @@ To learn more about running transactions using CLI, [see the docs](https://devel
 
 Let's now read the balance of the newly created account (`test-acct`) to check it's zero.
 
-Create this new script file `cadence/scripts/ReadVaultBalance.cdc`:
+Create this new script file `cadence/scripts/get_fooToken_balance.cdc`:
```

Comment:
I sticking with snake case would be more consistent
```suggestion
Create this new script file `cadence/scripts/get_foo_token_balance.cdc`:
```

Reply:
good idea. I'll update all the txs and scripts to have that